.github/workflows/build.yml

```diff
@@ -9,24 +9,35 @@ on:
 jobs:
   generate:
     runs-on: ubuntu-latest
-
     strategy:
       matrix:
         node-version: [12.x]
-
     steps:
     - uses: actions/checkout@v1
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: npm install, build and lint
-      run: |
-        npm ci
-        npm start
-        npm run lint
+    - run: npm ci
+    - name: Generate types
```

Comment:
It would be great to add build step names for each command (install deps, lint types)

Reply:
Yeah, they will just default to the command names, some of which are pretty clear, but down to make this consistent.